spec/features/idv/inherited_proofing/verify_wait_step_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+feature 'inherited proofing verify wait', :js do
+  include InheritedProofingWithServiceProviderHelper
+
+  before do
+    allow_any_instance_of(ApplicationController).to receive(:analytics).and_return(fake_analytics)
+    allow(IdentityConfig.store).to receive(:va_inherited_proofing_mock_enabled).and_return true
```

Comment:
add parens for clarity

```suggestion
    allow(IdentityConfig.store).to receive(:va_inherited_proofing_mock_enabled).and_return(true)
```

Reply:
Addressed.